docs/application/native/guides/machine-learning/machine-learning-service.md

```diff
@@ -4,7 +4,7 @@ The Machine Learning Service API provides utility interfaces for both AI develop
 
 The main features of the Machine Learning Service API include the following:
 
-- Provide AI pipelines, models, and resources as ML services via machine-learning-agent (daemon).
+- Provide AI pipelines, models, and resources as ML services via deviceMLOps.MLAgent (daemon).
```

Comment:
I recommend removing keyword daemon:
`as ML services via machine learning agent.`

Reply:
Thanks. Updated as you suggested.